test/elixometer_test.exs

```diff
@@ -135,6 +135,21 @@ defmodule ElixometerTest do
     assert subscription_exists "elixometer.test.counters.subscription"
   end
 
+  test "clearing a counter sets it to 0" do
+    update_counter("to_be_cleared", 1)
+    name = ["elixometer", "test", "counters", "to_be_cleared"]
+
+    wait_for_messages()
+    {:ok, [value: first_val]} = :exometer.get_value(name, :value)
```

Comment:
neat, I'll change it. I did this to match other tests but I think I prefer the condensed version